src/components/loginPage.jsx

```diff
@@ -69,12 +60,12 @@ const LoginPage = () => {
           <input
             type="password"
             name="password"
-            value={password}
+            value={loginDetails?.password}
```

Comment:
remove optional chaining... refactor to
```suggestion
            value={loginDetails.password}
```